### PR TITLE
CodeCamp #34

### PR DESCRIPTION
**Pull request by tianleiSHI**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

Please describe the motivation of this PR and the goal you want to achieve through this PR.
超级视客营活动-MMdetection-34文档翻译

## Modification
根据 en/user_guides/useful_tools and useful_hooks这两个文件的更新内容，将对应的中文文件进行翻译更新。大的调整有useful_tools.md中模型部署的步骤，另外修改了一些翻译细节
Please briefly describe what modification is made in this PR.

## BC-breaking (Optional)

Does the modification introduce changes that break the backward-compatibility of the downstream repos?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
3. If the modification has potential influence on downstream projects, this PR should be tested with downstream projects, like MMDet or MMCls.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

**Comment by BIGWangYuDong** [comment]
lint is failed, Please follow https://github.com/open-mmlab/mmcv/blob/master/CONTRIBUTING.md to fix it

File: docs/zh_cn/user_guides/useful_tools.md [hunk 1]
```diff
@@ -174,41 +176,30 @@ python tools/analysis_tools/coco_error_analysis.py \
...
+### 1. 安装 TorchServe
+假设你已经成功安装了包含`PyTorch`和`MMDetection`的`Python`环境，那么你可以运行以下命令来安装`TorchServe`及其依赖项。有关更多其他安装选项，请参考[快速入门](https://github.com/pytorch/serve/blob/master/README.md#serve-a-model)。
 
 ```shell
-python tools/deployment/mmdet2torchserve.py ${CONFIG_FILE} ${CHECKPOINT_FILE} \
---output-folder ${MODEL_STORE} \
---model-name ${MODEL_NAME}
+python -m pip install torchserve torch-model-archiver torch-workflow-archiver nvgpu
 ```
+**注意**: 如果你想在docker中使用`TorchServe`，请参考[torchserve docker](https://github.com/pytorch/serve/blob/master/docker/README.md)。
```
Comment (zytx121) [inline docs/zh_cn/user_guides/useful_tools.md:185]:
```suggestion
**注意**: 如果你想在docker中使用 `TorchServe`，请参考[torchserve docker](https://github.com/pytorch/serve/blob/master/docker/README.md)。
```

File: docs/zh_cn/user_guides/useful_tools.md [hunk 2]
```diff
@@ -174,41 +176,30 @@ python tools/analysis_tools/coco_error_analysis.py \
 
 如果你想使用 [`TorchServe`](https://pytorch.org/serve/) 搭建一个 `MMDetection` 模型服务，可以参考以下步骤：
 
-### 1. 把 MMDetection 模型转换至 TorchServe
+### 1. 安装 TorchServe
+假设你已经成功安装了包含`PyTorch`和`MMDetection`的`Python`环境，那么你可以运行以下命令来安装`TorchServe`及其依赖项。有关更多其他安装选项，请参考[快速入门](https://github.com/pytorch/serve/blob/master/README.md#serve-a-model)。
```
Comment (zytx121) [inline docs/zh_cn/user_guides/useful_tools.md:180]:
```suggestion
假设你已经成功安装了包含 `PyTorch` 和 `MMDetection` 的 `Python` 环境，那么你可以运行以下命令来安装`TorchServe`及其依赖项。有关更多其他安装选项，请参考[快速入门](https://github.com/pytorch/serve/blob/master/README.md#serve-a-model)。
```

File: docs/zh_cn/user_guides/useful_tools.md [hunk 3]
```diff
@@ -1,3 +1,5 @@
+除了训练和测试脚本，我们还在`tools/`目录下提供了许多有用的工具。
+
 ## 日志分析
 
 `tools/analysis_tools/analyze_logs.py` 可利用指定的训练log文件绘制 loss/mAP 曲线图，
```
Comment (zytx121) [inline docs/zh_cn/user_guides/useful_tools.md:5]:
```suggestion
`tools/analysis_tools/analyze_logs.py` 可利用指定的训练 log 文件绘制 loss/mAP 曲线图，
```

File: docs/zh_cn/user_guides/useful_tools.md [hunk 4]
```diff
@@ -1,3 +1,5 @@
+除了训练和测试脚本，我们还在`tools/`目录下提供了许多有用的工具。
```
Comment (zytx121) [inline docs/zh_cn/user_guides/useful_tools.md:1]:
```suggestion
除了训练和测试脚本，我们还在 `tools/` 目录下提供了许多有用的工具。
```

**Comment by zytx121** [comment]
Hi @tianleiSHI
Please provide an independent PR to modify the parameter of `super`. Let this PR focus on document modification.

**Pull request closed without merging**